package.json

```diff
@@ -54,11 +54,11 @@
     "prepare-release": "run-s reset-hard test cov:check doc:html version doc:publish"
   },
   "engines": {
-    "node": ">=12.9.0"
+    "node": ">=16.20.0"
```

Comment:
Not needed for now. We will drop versions prior 16 LTS in the next major release.